### PR TITLE
[READ THIS PR] Random maint: prototype

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
TL;DR: maint is now semi-random, with rooms appearing dependent on pre-sized templates. The current room pool is relatively small (80 rooms total), and is 85% maintenance rooms taken from pre-existing maps. Some rooms will contain special jobs that can only spawn if their room spawns, and some contain interesting gamer gear

Long version:
-retrofits box station for random maint (Currently, only box station has been converted, due to it having a decently-sized, varied maint, with space for all room types, without being so big the room pool will feel stale or even exhaust itself, as delta might risk.
~~-adds the Barber role, who can only be played if his room spawns in maintenance. also adds the maint job landmark to allow for similar roles in the future~~ (taken out for now)
-adds 70 new maint rooms. Most of these are ripped from preexisting maps

## Why It's Good For The Game
Maint is now more fun and interesting to explore, whilst also being harder to powergame. Adds gimmick potential that wont grow stale over several rounds

VISUAL AIDS:
Upper corner of box station, with randomized maint:
image
The barbershop:
image
One of the more interesting rooms: 
image
The bad corner of maint:
image

## Changelog
:cl:
add: straight razor and hand mirror
add: hair stylist sprays
add: chinese knockoff pda
del: Removed code for removed rooms
tweak: tweaked self surgery and razor interactions with mirrors
imageadd: Straight razor, hand mirror, and various spray sprites
add: 69 more random rooms
add: box station now has randomized maintenance
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
Mapdiffs is kill

**Comment by zeskorion** [comment]
yeah i added so many maps the bot died

**Comment by zeskorion** [comment]
travis you're drunk

**Comment by VictorVow** [comment]
@zeskorion is there a maint robotics room? Do you have any screenshots?

**Comment by zeskorion** [comment]
> @zeskorion is there a maint robotics room? Do you have any screenshots?

two

**Comment by ike709** [comment]
AI reported being able to see maint rooms without cameras in them.

**Comment by zeskorion** [comment]
> AI reported being able to see maint rooms without cameras in them.

huh, that's weird and i have no idea how that works... mmm

**Comment by VictorVow** [comment]
Proceeds to not post a screenshot...

Some rooms are missing floor tiles eg the gondola one didn't have plating underneath the airlock.

**Comment by zeskorion** [comment]
> Proceeds to not post a screenshot...
> 
> Some rooms are missing floor tiles eg the gondola one didn't have plating underneath the airlock.

that's an issue with the box map itself that i'm fixing rn. doors are not part of the rooms.
